cfgov/tccp/jinja2/tccp/card.html

```diff
@@ -418,6 +430,9 @@ <h2 class="h3 u-mb15">
                         </dd>
                     {% endif %}
                 </dl>
+                <p>
+                    <a href="/ask-cfpb/how-does-my-credit-card-company-calculate-the-amount-of-interest-i-owe-en-51/">Learn more about how credit card interest is calculated</a>.
```

Comment:
It feels a little weird not to include the closing period in the link when this is a full sentence, as opposed to a sentence fragment:

<img width="426" alt="image" src="https://github.com/cfpb/consumerfinance.gov/assets/654645/320c0381-e2d1-4367-bf22-24fa81dc3a58">

Is this our convention?

Reply:
I'm not sure what our convention is here. I suspect we don't have a consistent style for this, but I'll look around and see.